src/ethproto/w3wrappers.py

```diff
@@ -255,29 +359,31 @@ def eth_function(*args):
 
     def normalize_receipt(self, wrapper, receipt):
         if W3_TRANSACT_MODE == "defender-async":
-            return receipt  # Don't do anything because the receipt it's just a dict of not-yet-mined tx
+            return receipt  # Don't do anything because the receipt is just a dict of not-yet-mined tx
         return ReceiptWrapper(receipt, wrapper.contract)
 
     def _handle_exception(self, err):
         if str(err).startswith("execution reverted: "):
-            raise RevertError(str(err)[len("execution reverted: "):])
+            raise RevertError(str(err)[len("execution reverted: ") :])
         super()._handle_exception(err)
 
+    def _get_msg_args(self):
+        # TODO: This needs to be zero for test runs, but forcing it to zero like this will break other clients
+        return {"gasPrice": 0}
```

Comment:
Shall we merge this as is and then find a better solution?

Ideally this would be handled at the provider level or on a pytest fixture for tests.